ch02_Tactiek.md

```diff
@@ -996,7 +996,7 @@ impact hebben op de kwaliteit; op zijn minst zal de kwaliteit daardoor
 sterk verschillen per semantische standaard.
 
 <aside class="example" title="Floricode">
-[Floricode](https://www.floricode.com/) heeft dit kwaliteitsinstrument 
+<a href="https://www.floricode.com/">Floricode</a> heeft dit kwaliteitsinstrument 
```

Comment:
```suggestion

[Floricode](https://www.floricode.com/) heeft dit kwaliteitsinstrument
```
Vanaf een lege regel werkt de Markdowndetectie weer, zie bijvoorbeeld in [Voorbeeld 6](https://logius-standaarden.github.io/BOMOS-Verdieping/#example-template-voor-beheerdocument-en-rekenmodel-van-geonovum):
https://github.com/Logius-standaarden/BOMOS-Verdieping/blob/7fe0751a1ee38463181e06b3504a3a7b8cb84d4c/ch01_Strategie.md?plain=1#L525

Reply:
Dit had ik eerst geprobeerd, maar dat werkte volgens mij niet. Ik zal het nogmaals proberen en kijken of de publicatie preview hem wel pakt.

Reply:
Geen idee waarom het eerst niet leek te werken, maar [Floricode](https://logius-standaarden.github.io/Publicatie-Preview/BOMOS-Verdieping/TimvdLippe-patch-1/#example-floricode) en [GWSW server](https://logius-standaarden.github.io/Publicatie-Preview/BOMOS-Verdieping/TimvdLippe-patch-1/#example-validatie-van-gwsw) links doen het idd weer.